### DISCUSSION TITLE
Chose preset for bulk rna-seq

### DISCUSSION THREAD

**Discussion by EvgeniiaAlekseeva** [milaboratory/mixcr#1254] [Q&A] [answered]
Dear MiXCR team,

we are running the new version of mixcr on bulk rna-seq data and testing different presets to find the best suiting for our purposes. We reconstruct BCR repertoires from B cell lymphoma samples, so usually even in the bulk rna-seq library the full length or almost the full length of malignant clonotype is covered. For now the run with [rnaseq-cdr3 preset](https://mixcr.com/mixcr/reference/overview-analysis-overview/?h=cdr3) goes well, however it reconstructs only cdr3 parts. According the description in [your manual
](https://mixcr.com/mixcr/reference/overview-built-in-presets/?h=rna+seq+full+length#rna-seq-data) the command below is perfect for our goals:

mixcr analyze rna-seq \
    --species hsa \
    raw/CD8T_REH_4h_rep1.fastq.gz \
    results/CD8T_REH_4h_rep1

but unfortunately such run falls down with the error that there is no preset with such name. What can be the reason for this? May be it is the mistake in a preset's name?

With best wishes,
Evgenia Alekseeva

**Comment by mizraelson** [comment, marked as answer]
Hi Evgenia,
Can you please share the MiXCR version that you use. `rna-seq` preset is only available starting v4.4

**Reply by EvgeniiaAlekseeva** [reply]
We used 4.3.2, it seems that is the answer, thanks!
